ziti/cmd/edge/quickstart.go

```diff
@@ -140,13 +141,19 @@ func (o *QuickstartOpts) run(ctx context.Context) {
 		routerName = routerNameFromEnv
 	}
 
+	if o.AlreadyInitialized {
+		logrus.Warn("deprecated option --already-initialized has no effect and is internally implied by --home on subsequent runs")
```

Comment:
just remove it entirely. ziti edge quickstart is not widely adopted/promoted yet. but, do update the changelog